content/docs/api-reference/dvcfs.md

```diff
@@ -0,0 +1,66 @@
+# DvcFileSystem
+
+DvcFileSystem provides a pythonic file interface (aka
+[fsspec](https://filesystem-spec.readthedocs.io/)) for a DVC repo and provides a
+read-only filesystem-like operations like `ls`, `du`, `glob`, `get`, `download`,
+etc.
+
+DvcFileSystem provides a single view of all the files/directories in your
+repository, be it Git-tracked or DVC-tracked, or untracked (in case of a local
+repository). DvcFileSystem is smart to reuse the files in your cache directory
+(if present) and can otherwise stream/fetch the files from your default remote.
+
+### Basic Usage
+
+```py
+>>> from dvc.fs.dvc import _DvcFileSystem as DvcFileSystem
+# opening a local repository
+>>> fs = DvcFileSystem("/path/to/local/repository")
+# opening a remote repository
+>>> remote_fs = DvcFileSystem("https://github.com/iterative/example-get-started.git", rev="main")
+```
```

Comment:
```suggestion
>>> remote_fs = DvcFileSystem(
    "https://github.com/iterative/example-get-started.git", 
    rev="main")
```
```
The `rev` arg gets cropped in the block and it's not visible without scrolling. I think its important to show it given that it's not present in the fsspec API

Reply:
Thanks, I think I'll split them into two lines rather than indenting it.